Library/Homebrew/cmd/update-report.rb

```diff
@@ -127,15 +128,14 @@ def output_update_report
       updated = true
 
       old_tag = Settings.read "latesttag"
-
       new_tag = Utils.popen_read(
         "git", "-C", HOMEBREW_REPOSITORY, "tag", "--list", "--sort=-version:refname", "*.*"
       ).lines.first.chomp
```

Comment:
How about something like this to get a list of tags in between the current and previous tag:

```suggestion
      new_tags = Utils.popen_read(
        "git", "-C", HOMEBREW_REPOSITORY, "tag", "--list", "--sort=-version:refname", "*.*"
      ).lines.each(&:chomp)
      new_tags = new_tags.split(old_tag).first if old_tag.present? && new_tags.include?(old_tag)
      new_tag = tags.first
```

Reply:
See https://github.com/Homebrew/brew/pull/13482#discussion_r913220604